plugins/modules/redhat_subscription.py

```diff
@@ -70,6 +70,10 @@
         description:
             - Specify an HTTP proxy hostname.
         type: str
+    server_proxy_scheme:
+        description:
+            - Specify an HTTP proxy scheme, e.g. C(http) or C(https).
+        type: str
```

Comment:
```suggestion
        type: str
        version_added: 6.2.0
```

Reply:
ah thanks, I did not add a version as I was not sure about it

Reply:
The galaxy.yml file in the `main` branch usually contains the version of the next minor/major version that accepts new features. But it's totally OK if you don't specify it or just guess, I'll add a suggestion like the above one then :)